phobos/io/representation.py

```diff
@@ -383,7 +385,7 @@ def __init__(self, geometry=None, material=None, material_: Material = None, ori
         self.geometry = _singular(geometry)
         material_ = _singular(material_)
         material = _singular(material)
-        if type(material) == str:
+        if type(material) == str and hasattr(material_, "original_name"):
```

Comment:
We should here rather add the assertion in the following line that (if material is string) material_ is of type Material